autoload/crystal/indent.vim

```diff
@@ -1,162 +1,885 @@
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
I think cpo-save is no longer necessary. It was necessary for ancient Vim or setting `compatible` option (please see `:help use-cpo-save`). But with such an ancient Vim, this plugin would not work (since other features are also not supported). `compatible` option is as well.

Reply:
For now, I have removed all references to cpo_save.